lua/nvim-tree/view.lua

```diff
@@ -133,7 +133,6 @@ local function setup_tabpage(tabpage)
 end
 
 local function set_window_options_and_buffer()
-  pcall(vim.cmd, "buffer " .. M.get_bufnr())
```

Comment:
Can i broke something with this deletion? This one also used in `open_win`, maybe should make separate functions `set_buffer` and `set_window_options`?